Codes/SlippiReplays/EXIPacket.h

```diff
@@ -9,7 +9,12 @@ enum EXICommand
 {
     START_REPLAYS_STRUCT = 19,
     REPLAYS_STRUCT = 20,
-    ENDGAME = 21
+    ENDGAME = 21,
```

Comment:
Maybe out of scope for this review, but these values are shared between dolphin / game, so belong in the exi-structures repo IMO